Requirements/base.txt

```diff
@@ -0,0 +1,8 @@
+dj-database-url==0.4.2
+Django==1.11
+flake8==3.5.0
```

Comment:
This dep is for dev env, but I would say you don't need this, since it's managed separately by pre-commit tool